CRM/Utils/Check.php

```diff
@@ -209,7 +212,7 @@ public static function checkAll($max = FALSE) {
 
     foreach ($messages as $key => $message) {
       $hush = self::checkHushSnooze($message);
-      $message->setVisible(!$hush);
+      $messages[$key]->setVisible(!$hush);
```

Comment:
Oh I see. This change isn't actually necessary because $message is an object so it works either way.